doc(release): Prepare release notes for modules 22.10 (#2882)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.10.14

Release date: `December 21, 2023`

#### Enhancements

> - [Viewer] Fixed status as font color for output widget.

Reviewer: ```suggestion
- [Viewer] Fixed an issue with the "status as font color" option in the Output widget.
```

#### Bug fixes

> - [Editor] Removed border and fill colors parameters for Centreon Resources shapes.

Reviewer: ```suggestion
- [Editor] Removed border and fill colors parameters for Centreon resources shapes.
```

## Centreon MBI

### 22.10.7

Release date: `December 8, 2023`

#### Bug fixes

- [MBI] Updated dependency.

### 22.10.6

Release date: `September 14, 2023`

#### Bug fixes

> - [Packaging] Fixed a regression in centreon-bi-engine packaging that caused a broken dependency situation with Java.

Reviewer: ```suggestion
- [Packaging] Fixed a regression in the packaging of centreon-bi-engine that caused a broken dependency situation with Java.
```
